content/backstage/plugins/sonarqube.md

```diff
@@ -63,16 +62,34 @@ gettingStarted:
         target: https://your.sonarqube.instance.com/api
         allowedMethods: ['GET']
         headers:
-          Authorization: # Content: 'Basic base64("<api-key>:")' <-- note the trailing ':' # Example: Basic bXktYXBpLWtleTo=
-            $env: SONARQUBE_AUTH_HEADER
+          Authorization: Basic ${SONARQUBE_AUTH}
 
       sonarQube:
         baseUrl: https://your.sonarqube.instance.com
 
+  - intro: Get and provide SONARQUBE_AUTH/SONARCLOUD_AUTH as env variables (see Notes for more details)
```

Comment:
Do the notes actually have more details? It looks like the notes are more related to the environment variable.